connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -273,6 +274,21 @@ message StatFunction {
   message Summary {
     repeated string statistics = 1;
   }
+
```

Comment:
shall we make the proto message a bit more flattened? It looks to me that the only shared part between `Summary` and `CrossTabulate` is the input relation. Shall we simply have 2 messages?
```
message StatsSummary {
  Relation input = 1;
  ...
}
message StatsCrossTabulate {
  Relation input = 1;
  ...
}
```

Reply:
Or a different strategy is shared a field named arguments but the server side will do more to differentiate the inputs for Summary, CrossTab, etc. This seems to be bad.

Separating into different messages makes sense with existing example about  Summary and CrossTab

Reply:
ok, let me flatten summary first

Reply:
+1 to make it flattened or simpler.